tutorials/dataframe/df002_dataModel.py

```diff
@@ -76,7 +77,7 @@
 
 getPt_code ='''
 using namespace ROOT::VecOps;
```

Comment:
I don't think this is needed anymore

Reply:
I did `namespace ROOT` in this file, since twice there is RvecD (lines 80 and 90). I will use `ROOT::RvecD` instead.

Reply:
let me clarify, I don't think `using namespace ROOT::VecOps` is needed -- `using namespace ROOT` is.